parts/dcos/dcosmastervars.t

```diff
@@ -26,6 +26,7 @@
         {{end}}
       ],
 {{end}}
+    "orchestratorVersion": "{{.OrchestratorProfile.OrchestratorVersion}}",
```

Comment:
Here in agentWindowsVersion can we tag with whether it is a Windows or a Linux agent?

Reply:
OS type has its own field